src/components/entity/ha-entity-picker.ts

```diff
@@ -54,6 +58,8 @@ export class HaEntityPicker extends LitElement {
 
   @property() public helper?: string;
 
+  @property() public createDomains?: string[];
```

Comment:
This is an array now, but only the first value seems to be used?

Reply:
For the moment yes. I wasn't sure quite sure which way this would go, or if we might at some point want to have multiple options in some cases (like if you could create any type of sensor?). 

Was sort of trying to future proof a bit but can remove if we want to keep it simpler.

Reply:
I'm fine with future proofing, but then I think we should also implement it working as an array, or just have the code clear and use:

```ts
const createDomain = createDomains[0];
```

Reply:
Modified the create dialog to support passing multiple domains instead of just one or undefined.

Reply:
But the entity picker still only shows the first item?

Reply:
Hmm I think if there are mutliple types it has a generic "Add a helper" item, and then the dialog box that pops up has a picker for the types you can choose.

Reply:
But that's somewhat moot as it's not currently possible.

Reply:
I would have added 1 item per domain in the entity picker for now, and not changed the add helper dialog

Reply:
> I would have added 1 item per domain in the entity picker for now, and not changed the add helper dialog

Is that a request for a change (are we holding the review for this)?